examples/solids/index.rst

```diff
@@ -322,27 +327,27 @@ It is crucial to distinguish between the current and reference element in the To
 
     \int_{\Omega}{\boldsymbol{v} \cdot \left(\nabla_X \cdot \boldsymbol{P} + J \rho \boldsymbol{g}\right)} dV = \boldsymbol{0}
 
-By Integration by part we arrive at the weak form:
+Integrating by parts, we arrive at the weak form:
 
 .. math::
 
     \int_{\Omega}{\nabla_X \boldsymbol{v} \colon \boldsymbol{P}}dV =
   - \int_{\Omega}{\boldsymbol{v} \cdot \rho_0 \boldsymbol{g}}dV
   - \int_{\partial \Omega}{\boldsymbol{v} \cdot \boldsymbol{P} \cdot \hat{\boldsymbol{N}}}dA
     
-where :math:`\boldsymbol{P} \cdot \hat{\boldsymbol{N}}` is a prescribed boundary written in terms of reference
-configuration.
+where :math:`\boldsymbol{P} \cdot \hat{\boldsymbol{N}}` is a prescribed boundary
+condition written in terms of the reference configuration.
 
-The constitutive law in indicial notation is given by:
+For simplicity, we express here the constitutive law in indicial notation:
 
 .. math::
 
-   P_{iI} = F_{iB}S_{BI} 
+   P_{iI} = F_{iB}S_{BI}\, ,
```

Comment:
I'm just curious, but is this a rendering issue on your end? It's easy to forget this convention and it makes the math noisier to always have to throw in the extra bit of space `\,`. A raw comma looks fine to my eye.

Reply:
No it's just that especially when there are terms with subscripts in the equations, the comma or semicolon tend to be too close to them. Just a matter of aesthetics. Can be removed of course.

Reply:
Thanks, seems fair to add them (when we remember) after equations that end in subscript. No need to spend more time on this point.